src/BlitHelper.cpp

```diff
@@ -174,15 +191,14 @@ namespace D3D12TranslationLayer
         //
         // setup the RTV
         //
-        bool needsTempRenderTarget = (dstDesc.Flags & D3D12_RESOURCE_FLAG_ALLOW_RENDER_TARGET) == D3D12_RESOURCE_FLAG_NONE;
-        bool needsTwoPassColorConvert = (bSwapRBChannels && CD3D11FormatHelper::YUV(pSrc->AppDesc()->Format()));
+        bool needsTwoPassColorConvert = (bSwapRBChannels && CD3D11FormatHelper::YUV(srcFormat));

```

Comment:
I think I'd move this up and assert that it's false if we're doing an MSAA resolve.

Reply:
Ah good idea, I'll do that